rpm/cloud-info-provider-opennebula.spec

```diff
@@ -14,7 +14,6 @@ URL: https://github.com/EGI-Foundation/cloud-info-provider
 
 Requires: cloud-info-provider
 # OpenNebula-specific dependencies
-Requires: python-defusedxml
```

Comment:
It may be appropriate to still keep the dep here, as it's really required here.

Reply:
Maybe my comment is just irrlevant, feel free to ignore :)

Reply:
fixed and also added pyOpenSSL and requests that were missing